### PR TITLE
update internships banner

### PR DESCRIPTION
**Pull request by philspraydfe**
### Trello card

https://trello.com/c/QmfWqBt2

### Context

Need to update the banner wording on the internships page to highlight the application window is opening in January 2023.

### Changes proposed in this pull request

Updated banner header
Amended banner copy
Explained context of email signup
Changed button CTA to email signup

### CODE REVIEW

File: app/views/content/teaching-internship-providers.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ keywords:
       <span class="icon icon-warning" aria-hidden="true"></span>
     </div>
     <div>
-      <h2 class="heading-m">Applications now closed</h2>
+      <h2 class="heading-m">Apply from January 2023</h2>
       
-      <p>Applications for the Teaching Internships Programme in 2022 are now closed. Applications will reopen by early 2023.</p>
+      <p>Applications for the Teaching Internships Programme will open in January 2023. More information on how to apply will be available here, once applications open.</p>
```
Comment (EllieNodder) [inline app/views/content/teaching-internship-providers.md:43]:
```suggestion
      <p>Applications for the Teaching Internships Programme will open in January 2023. More information on how to apply will be available on this page, once applications open.</p>
```
Just a small suggestion - feel free to ignore!

**Review by EllieNodder — APPROVED** [review]
Once small suggestion, but overall, looks good to me!

File: app/views/content/teaching-internship-providers.md [hunk 2]
```diff
@@ -38,13 +38,13 @@ keywords:
       <span class="icon icon-warning" aria-hidden="true"></span>
     </div>
     <div>
-      <h2 class="heading-m">Applications now closed</h2>
+      <h2 class="heading-m">Apply from January 2023</h2>
       
-      <p>Applications for the Teaching Internships Programme in 2022 are now closed. Applications will reopen by early 2023.</p>
+      <p>Applications for the Teaching Internships Programme will open in January 2023. More information on how to apply will be available on this page, once applications open.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/teaching-internship-providers.md:43]: @philspray I'm not sure we should talk about applying to the 'Teaching Internships Programme' - you don't apply to the programme but to individual schools offering internships. I also think 'programme' is very internal language?

Could we say something like 'You will be able to apply for a teaching internship from January 2023'?

File: app/views/content/teaching-internship-providers.md [hunk 3]
```diff
@@ -38,13 +38,13 @@ keywords:
...
 
-      <p>If you’re interested in spending some time in a classroom, you can still get school experience.</p>
+      <p>Sign up for our emails and we will send you helpful advice and insights about getting into teaching. If you are eligible for the Internships Programme, we will also let you know when applications open.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/teaching-internship-providers.md:45]:
```suggestion
      <p>Sign up for our emails and we will send you helpful advice and insights about getting into teaching. If you are eligible for a teaching internship, we will also let you know when applications open.</p>
```

**Review by gemmadallmandfe — COMMENTED** [review]
Thanks @philspray! Looks good - just a couple of minor suggestions from me

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged when the internships team have seen it

**Pull request merged**